airflow/contrib/utils/parameters.py

```diff
@@ -16,6 +16,7 @@ class XComParameter(object):
     will be sent... somehow to be determined later. Options include delimited
     GET params, multiple GET params, or POST body.
     """
+
     def __init__(self, task_ids, xcom_key='return_value'):
```

Comment:
I was expecting the callback to be passed in here, as a transformation that would be applied to the received value. I'm not sure how to pass in the callback in the way you've defined it.

Reply:
like this https://github.com/TriggerMail/airflow-dag/pull/1857/files